tests/linux_kernel/kmod/drgn_test.c

```diff
@@ -385,6 +386,45 @@ static int drgn_test_slab_init(void)
 	return 0;
 }
 
+// kthread for stack trace
+
+static struct task_struct *drgn_kthread;
+
+static int __attribute__((optimize("O0"))) drgn_kthread_fn(void *arg)
```

Comment:
Lol at `O0`. I guess I can't think of a better way to do it.

Reply:
Haha yeah, I'm sorry for that, but it did seem to do the trick.

I guess I don't actually test whether the contents of the variables are present, so the attribute could be dropped. DWARF should have the names of the variables and parameters regardless. I had it around in the userspace version of my test so that I could also lookup the values. But I don't know if it matters much either way.